core/src/main/java/org/eclipse/hono/util/Constants.java

```diff
@@ -77,6 +77,10 @@
      * The type of the sigfox protocol adapter.
      */
     public static final String PROTOCOL_ADAPTER_TYPE_SIGFOX = "hono-sigfox";
+    /**
+     * The type of the protocol adapter which actually denotes the device registry.
+     */
+    public static final String PROTOCOL_ADAPTER_TYPE_DEVICE_REGISTRY = "hono-device-registry";
```

Comment:
Yes, see the discussion we already had on this: https://github.com/eclipse/hono/pull/2094#discussion_r469994430

Since the application property name whose value is set to the contents value is _orig_adapter_ I thought it doesn't make sense to "hide" this in the name...